### PR TITLE
PRL-5665

### PR DESCRIPTION
**Pull request by sealDebolina**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-5665
### Change description ###

hide train track

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/common/task-list/content.ts [hunk 1]
```diff
@@ -140,6 +140,7 @@ export const generateContent: TranslationFn = content => {
   const caseData = request.session.userCase;
   const partyType = request.params.partyType;
   const _isRepresentedBySolicotor = isRepresentedBySolicotor(caseData, request.session.user.id);
+  const hideTraintrack = false;
```
Comment (vivek-sekhar) [inline src/main/steps/common/task-list/content.ts:143]: Set up a feature toggle in LD to control.

File: src/main/steps/common/task-list/content.test.ts [hunk 2]
```diff
@@ -204,33 +204,33 @@ describe('testcase for tasklist', () => {
...
-        label: 'Gwrandawiadau <br/>a<br/> gorchmynion llys',
-        statusBarClassName: '',
-      },
-      {
-        ariaLabel: 'Cam achos wedi’i gau heb ddechrau eto',
-        label: 'Achos wedi’i <br/>gau',
-        statusBarClassName: '',
-      },
-    ],
+    // progressBar: [
```
Comment (vivek-sekhar) [inline src/main/steps/common/task-list/content.test.ts:207]: Dont comment out code. Handle through config with LD value

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/common/task-list/content.test.ts [hunk 3]
```diff
@@ -82,29 +82,29 @@ describe('testcase for tasklist', () => {
...
-        statusBarClassName: '',
-      },
-      {
-        ariaLabel: 'Hearings and court orders stage is not yet started',
-        label: 'Hearings and<br/> court orders',
-        statusBarClassName: '',
-      },
-      { ariaLabel: 'Case closed stage is not yet started', label: 'Case closed', statusBarClassName: '' },
-    ],
+    // progressBar: [
```
Comment (yogendraU) [inline src/main/steps/common/task-list/content.test.ts:85]: please remove commented line of code.

File: src/main/steps/common/task-list/content.ts (hunk 1 above)
Comment (sealDebolina) [reply src/main/steps/common/task-list/content.ts]: done

File: src/main/steps/common/task-list/content.test.ts (hunk 2 above)
Comment (sealDebolina) [reply src/main/steps/common/task-list/content.test.ts]: done

File: src/main/steps/common/task-list/content.test.ts (hunk 3 above)
Comment (sealDebolina) [reply src/main/steps/common/task-list/content.test.ts]: added LD flag, so we don't need to comment out portion of the code for future

File: src/main/app/utils/featureToggles.ts [hunk 4]
```diff
@@ -49,6 +49,12 @@ export class FeatureToggles {
     return isTestingSupportEnabled;
   }
 
+  async isCitizenTrainTrackFeatureEnabled(): Promise<boolean> {
```
[the remark this answers is not in the document]
Comment (sealDebolina) [reply src/main/app/utils/featureToggles.ts]: done

File: src/main/steps/common/task-list/content.ts [hunk 5]
```diff
@@ -163,7 +163,9 @@ export const generateContent: TranslationFn = content => {
       },
     ],
     partyName: getPartyName(caseData, partyType, request.session.user),
-    progressBar: getProgressBarConfig(caseData, partyType, content.language, request.session.user),
+    progressBar: request.session.citizenTrainTrackFeature
```
[the remark this answers is not in the document]
Comment (sealDebolina) [reply src/main/steps/common/task-list/content.ts]: done

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**